lib/src/components/NxGlobalSidebar/NxGlobalSidebar.scss

```diff
@@ -94,7 +95,7 @@ $-separator-color: var(--nx-swatch-indigo-40);
   @include nx-container-helpers.container-horizontal;
   @include nx-text-helpers.truncate-ellipsis;
 
-  border: 1px solid transparent;
+  border: 2px solid transparent;
```

Comment:
This is also increasing the overall height of these elements, and should be dealt with through a padding decrease.